crates/ubrn_cli/src/codegen/templates/NativeCodegenTemplate.ts

```diff
@@ -0,0 +1,11 @@
+// Generated by uniffi-bindgen-react-native
+import type { TurboModule } from 'react-native';
+import { TurboModuleRegistry } from 'react-native';
+
+export interface Spec extends TurboModule {
+  multiply(a: number, b: number): number;
```

Comment:
Same here, multiply is temporary right?

Reply:
yes, multiply is temporary. I should add some TODOs.